IndicoV2.IntegrationTests/Utils/IndicoTestContainerBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Indico;
+using IndicoV2.DataSets;
+using IndicoV2.IntegrationTests.Utils.DataHelpers.Submissions;
+using IndicoV2.Submissions;
+using IndicoV2.Workflows;
+using Unity;
+using V1Client = Indico.IndicoClient;
+using V2Client = IndicoV2.IndicoClient;
+
+namespace IndicoV2.IntegrationTests.Utils
+{
+    internal class IndicoTestContainerBuilder
+    {
+        private string BaseUrl => Environment.GetEnvironmentVariable("INDICO_HOST") ?? "https://app.indico.io";
```

Comment:
Do we want the integration tests to fail to prod if the env var is missing or would it make more sense to fail to a lower env?

Reply:
definitely - removed